neonvm/apis/neonvm/v1/virtualmachine_webhook.go

```diff
@@ -145,6 +145,11 @@ func (r *VirtualMachine) ValidateUpdate(old runtime.Object) (admission.Warnings,
 		}
 	}
 
+	// allow to change CPU scaling mode only if it's not set
+	if r.Spec.CpuScalingMode != nil && before.Spec.CpuScalingMode != nil && *r.Spec.CpuScalingMode != *before.Spec.CpuScalingMode {
```

Comment:
question: This still allows (non-nil) -> (nil), right? is that intentional?

Reply:
Nope, it is a great catch in the logic flaw, thanks for pointing out